src/static/css/utilities.css

```diff
@@ -16,29 +14,26 @@
 }
 
 /* Gap */
-
-.gap-1 {
+.gap-s {
     gap: var(--space-s);
 }
 
-.gap-2 {
+.gap-m {
     gap: var(--space-m);
 }
 
 /* Height */
-
-.height-7 {
+.height-3x1 {
```

Comment:
Typo:

```suggestion
.height-3xl {
```

Reply:
I really don't know how I made this mistake :)